app/workers/gamedev_worker.rb

```diff
@@ -12,4 +12,7 @@ def perform
 		  p.save
 	    end
 	end
+  
+  Sidekiq::Cron::Job.create( name: 'RubyStackoverflow fetch questions every one minute', cron: '* * * * 5', klass: 'GamedevWorker')
```

Comment:
Line is too long. [131/80]<br>Inconsistent indentation detected.<br>Space inside parentheses detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.